src/NuGet.Core/NuGet.Build.Tasks/GetReferenceNearestTargetFrameworkTask.cs

```diff
@@ -248,19 +248,20 @@ private static NuGetFramework GetNuGetFramework(TargetFrameworkInformation targe
                     string.Empty :
                     targetFrameworkInformation._targetPlatformMoniker);
         }
-    }
-
-    internal class TargetFrameworkInformation
-    {
-        internal readonly string _targetFrameworkAlias;
-        internal readonly string _targetFrameworkMoniker;
-        internal readonly string _targetPlatformMoniker;
 
-        public TargetFrameworkInformation(string targetFrameworkAlias, string targetFrameworkMoniker, string targetPlatformMoniker)
+        private class TargetFrameworkInformation
```

Comment:
This was just moved to be a subclass since I had to make `NuGet.Build.Tasks` internals visible to `NuGet.Build.Tasks.Console.Test`